docs/docs/web-support.md

```diff
@@ -0,0 +1,23 @@
+---
+id: web-support
+title: Web Support
+sidebar_label: Web Support
+---
+
+Since
+[2.0.0-alpha.7](https://github.com/software-mansion/react-native-reanimated/releases/tag/2.0.0-alpha.7)
+has been released it is possible to launch reanimated 2 in a web browser. For that case all of the functionalities are implemented purely in javascript, hence the efficiency of the animations might drop.
```

Comment:
```suggestion
release it's' possible to launch reanimated 2 in a web browser. On the web version, all of the functionalities are implemented purely in javascript and the efficiency of the animations might drop.
```

Reply:
Well I'm not sure if those changes are good:
`release it's'` - that second apostrophe look like an error, but as for those shortcuts it's ok
`On the web version` - `web` word repetition, not that it's that bad, just not a good change for me
`and` - `hence` just suits better I think

Reply:
1. > release it's' - that second apostrophe look like an error
Yeah, fixed

2. & 3. Good point, what you think about "All of the web functionalities are implemented purely in javascript which may decrease the efficiency of animations."

Reply:
Doesn't really matter to me, basically you could rewrite all of the content since every piece of information can be passed using alternative words.